regression/python/classes/main.py

```diff
@@ -1,16 +1,21 @@
 class MyClass:
     def __init__(self, value: int):
-        self.data:int = value
+        self.x:int = value
+
+    def foo(self):
+        pass
 
 
 obj1 = MyClass(5)
-assert(obj1.data == 5)
-obj1.data = 10
-assert(obj1.data == 10)
-obj1.data = 20
-assert(obj1.data == 20)
+assert(obj1.x == 5)
```

Comment:
I recommend keeping the original regression and creating a new one with these changes.